File: src/test/java/torrent/SolidTorrentsTest.java
package torrent;

import org.junit.jupiter.api.Test;
import utilities.HttpHelper;

import java.util.List;

import static org.junit.Assert.assertTrue;

class SolidTorrentsTest {

    @Test
    void shouldFindTorrents() {
        // Given
        // When
        List<Torrent> torrentList = new SolidTorrents(new HttpHelper()).searchTorrents("planet");
        // Then
        assertTrue(torrentList.size() > 0);
    }
}

File: src/test/java/boat/torrent/SolidTorrentsTest.kt
package boat.torrent

import boat.utilities.HttpHelper
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class SolidTorrentsTest {
    @Test
    fun shouldFindTorrents() {
        // Given
        // When
        Assertions.assertDoesNotThrow<List<Torrent>> { SolidTorrents(HttpHelper()).searchTorrents("planet") }
    }
}
